applications/minotari_console_wallet/src/automation/commands.rs

```diff
@@ -271,6 +321,18 @@ pub async fn coin_split(
     Ok(tx_id)
 }
 
+pub fn sign_message(private_key: String, challenge: String) -> Result<Signature, CommandError> {
```

Comment:
```suggestion
pub fn sign_message(private_key: KeyID, challenge: String) -> Result<Signature, CommandError> {
```
I think its safer to do this via a keyId approach